src/client/app/types/items.ts

```diff
@@ -104,3 +105,15 @@ export enum TrueFalseType {
 	true = 'yes',
 	false = 'no'
 }
+
+// user default values
+export const userDefaults = {
+	username: '',
+	password: '',
+	confirmPassword: '',
+	note: '',
+	role: UserRole['Select Role'],
```

Comment:
This should be set up so it is translated when shown to the user.

Reply:
What do you mean by this?

Reply:
We made this a default value enum, how would we go about doing this?

Reply:
I think I figured out a way and I'll implement it.